guides/source/getting_started.md

```diff
@@ -242,262 +241,740 @@ different _actions_. Each action's purpose is to collect information to provide
+because we saw it ourselves, but also because this line contains `200 OK`,
+which is the HTTP status code for successful responses. The rest of the
+information on this line is performance metrics.
+
+We have now finished rendering our first view. We created two routes that go to
+the same action -- `GET /` and `GET /articles`. That action is the `index`
+action in the `ArticlesController`. When we make a request to either of these
+paths, the `app/views/articles/index.html.erb` view is rendered, wrapped in the
+`app/views/layouts/application.html.erb` layout.
+
 TIP: For more information about routing, refer to [Rails Routing from the Outside In](routing.html).
 
-Getting Up and Running
-----------------------
+## Adding Articles
+
+So far, we have seen routes, controllers, actions and views within our Rails
+application. All of these are conventional parts of Rails applications and it
+is done this way to follow the MVC pattern. The MVC pattern is an application
+design pattern which makes it easy to separate the different responsibilities
+of applications into easy to reason about pieces.
 
-Now that you've seen how to create a controller, an action, and a view, let's
-create something with a bit more substance.
+So with "MVC", you might guess that the "V" stands for "View" and the "C"
+stands for controller, but you might have trouble guessing what the "M" stands
+for. This next section is all about that "M" part, the _model_.
 
-In the Blog application, you will now create a new _resource_. A resource is the
-term used for a collection of similar objects, such as articles, people, or
-animals.
-You can create, read, update, and destroy items for a resource and these
-operations are referred to as _CRUD_ operations.
+### Generating an Article model
 
-Rails provides a `resources` method which can be used to declare a standard REST
-resource. You need to add the _article resource_ to the
-`config/routes.rb` so the file will look as follows:
+A model is a class that is used to represent data in our application. In a
+plain-Ruby application, you might have a class defined like this:
 
 ```ruby
-Rails.application.routes.draw do
-  get 'welcome/index'
+class Article
+  attr_reader :title, :body
 
-  resources :articles
+  def initialize(title:, body:)
+    @title = title
+    @body = body
+  end
+end
+```
+
+In a Rails application, models are used for more than just representing data.
+Models are also used to interact with the application's database. Models use a
+part of Rails called Active Record to interact with the database of the
+application.
+
+In this section, we're going to use a model to put data into our database and
+to pull that data back out, using methods that are provided to us from Active
+Record.
+
+To start with, we're going to need to generate a model. We can do that with the
+following command:
+
+```bash
+$ bin/rails g model article title:string body:text
```

Comment:
I think it's clearer for beginners to avoid shortcuts and use `generate` instead of `g`

Reply:
I think explaining this earlier on will be better than expanding all `g` to `generate`: 431219185c